cves/kernel/CVE-2020-15780.yml

```
@@ -151,10 +158,15 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This bug was originally discovered by an independent security researcher. They 
+    discovered it by following a similar process used to exploit a different
+    vulnerability (not reference by name). Using these previous known tricks, they 
+    discovered that you can inject an APCPI table and suggested the fix of checking 
```

Misspelling "APCPI" should be "ACPI"
```suggestion
    discovered that you can inject an ACPI table and suggested the fix of checking 
```